### PR TITLE
0.2/143 ui identifier logged in

### PR DESCRIPTION
**Pull request by gingerbreadpup**
Some static content here in the new profile bar, will need to hook it up later. Still a work in progress but maybe getting towards something now.

:sleeping:

### CODE REVIEW

File: src/components/Profile.vue [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<template id="profile">
+
+    <div class="profile-bar">
+        <div><strong>2</strong> vouchers added</div>
+        <div>Logged in as <strong>P &amp; J Millward</strong></div>
```
Comment (charlesstrange2) [inline src/components/Profile.vue]: Worth adding an clicklink to accounts page on that name?

File: src/components/Toggle.vue [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<template id="toggle">
```
Comment (charlesstrange2) [inline src/components/Toggle.vue]: Toggle? toolbar perhaps?

**Review by charlesstrange2 — APPROVED** [review]
works, but just have some nomenclature ideas.
:sheep::it:
Ship it

File: src/components/Toggle.vue (hunk 2 above)
Comment (gingerbreadpup) [reply src/components/Toggle.vue]: Yeah, that works!

**Pull request merged**